src/view/shell/index.web.tsx

```diff
@@ -17,18 +17,21 @@ import {BottomBarWeb} from './bottom-bar/BottomBarWeb'
 import {useNavigation} from '@react-navigation/native'
 import {NavigationProp} from 'lib/routes/types'
 import {useAuxClick} from 'lib/hooks/useAuxClick'
+import {useShellState} from '#/state/shell'
 
 const ShellInner = observer(function ShellInnerImpl() {
   const store = useStores()
+  const {isDrawerOpen, setIsDrawerOpen} = useShellState()
   const {isDesktop, isMobile} = useWebMediaQueries()
   const navigator = useNavigation<NavigationProp>()
   useAuxClick()
 
   useEffect(() => {
     navigator.addListener('state', () => {
+      setIsDrawerOpen(false)
       store.shell.closeAnyActiveElement()
```

Comment:
Need to do this here instead of in a centralized `closeAnyActiveElement`. We will migrate this later on.